packages/react/src/components/Tree/Tree.tsx

```diff
@@ -60,14 +78,47 @@ class Tree extends UIComponent<ReactProps<TreeProps>> {
     accessibility: defaultBehavior,
   }
 
+  static autoControlledProps = ['activeIndex']
+
+  getInitialAutoControlledState({ defaultActiveIndex, exclusive }) {
+    return {
+      activeIndex: defaultActiveIndex === undefined ? (exclusive ? -1 : []) : defaultActiveIndex,
+    }
+  }
+
+  computeNewIndex = index => {
+    const { activeIndex } = this.state
+    const { exclusive } = this.props
+
+    if (exclusive) return index === activeIndex ? -1 : index
+    // check to see if index is in array, and remove it, if not then add it
+    return _.includes(activeIndex, index) ? _.without(activeIndex, index) : [...activeIndex, index]
```

Comment:
```suggestion
    return _.includes(activeIndex as number[], index) ? _.without(activeIndex as number[], index) : [...activeIndex as number[], index]
```

As we apply checks on `exclusive`, TS doesn't know that `activeIndex` is `number`. @mnajdova do you have better ideas? 💭